### DISCUSSION TITLE
TezosToolkit is not a constructor

### DISCUSSION THREAD

**Discussion by NguyennThinh** [ecadlabs/taquito#2718] [Q&A]
I got an error 'TezosToolkit is not a constructor' when i try  'const Tezos = new TezosToolkit('https://ghostnet.ecadinfra.com');'

**Comment by hui-an-yang** [comment]
Might need more info to help you like what's your setup like module in tsconfig.json and how do you import TezosToolkit into your file?
Is this a frontend or backend project?
